tidb-cloud/import-sample-data.md

```diff
@@ -31,7 +31,9 @@ This document describes how to import sample data into TiDB Cloud via the UI. Th
 
     - **Data format**: select **SQL File**.
     - **Bucket URI**: enter the sample data URI `s3://tidbcloud-sample-data/data-ingestion/`
-    - **Role ARN**: enter `arn:aws:iam::801626783489:role/import-sample-access`
+    - **Bucket Access**: you can use either an Amazon access key or a Role ARN to access your bucket.
+        - **AWS Access Keys**: enter the access key ID and secret access key. For more information about configuring access keys, see [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
```

Comment:
Do we have the available AWS Access Keys for the sample data at s3://tidbcloud-sample-data/data-ingestion/? If not, we can tell users to skip this option for sample data?

Reply:
@okJiang PTAL

Reply:
The sample-data is not available by any Access Keys. I think this is a new feature we have not completed.

Reply:
updated